app/views/claim_mailer/update_after_three_weeks.text.erb

```diff
@@ -0,0 +1,11 @@
+Dear <%= @display_name %>,
+
+We're still reviewing your claim <%= @claim_description %>, which we received on <%= @claim.submitted_at %>. Claim reference: <%= @claim.reference %>.
+
+We'll notify you as your claim progresses. We do not expect any delays to the processing of your claim.
+
+You do not need to do anything.
+
+Regards,
+
+The claim additional payments for teaching team
```

Comment:
I'd probably capitalise `Claim Additional Payments for Teaching` here

Reply:
As per Slack chat, we should be "The Claim additional payments for teaching team"